R/render.r

```diff
@@ -27,7 +27,25 @@ render_page <- function(pkg = ".", name, data, path = "", depth = NULL, quiet =
     depth <- length(strsplit(path, "/")[[1]]) - 1L
   }
 
-  data$logo <- list(src = logo_path(pkg, depth = depth))
+  logo_path <- logo_path(pkg, depth = depth)
+
+  # Absolute URL for 404
+  if (!is.null(logo_path)) {
```

Comment:
Could we move this into `tweak_404_links()` by also tweaking `img` and then applying `tweak_404_links()` t to the whole page?

Reply:
such a good idea :smiley: